5_analyze_data/notebooks/KS-test_analysis/visualize_ks_test.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Visualize the results of the KS-test
+
+# In[1]:
+
+
+suppressPackageStartupMessages(library(ggplot2))
+suppressPackageStartupMessages(library(dplyr))
+
+
+# In[2]:
+
+
+output_fig_dir <- "figures"
+
+ks_test_fig <- file.path(output_fig_dir, "ks_test_cp_genotype.png")
+actin_distrib_fig <- file.path(output_fig_dir, "top_actin_feature_distrib.png")
+
+
+# In[3]:
+
+
+# Load and process KS-test data
+ks_file <- file.path("data", "nf1_kstest_two_sample_results.csv")
+ks_df <- readr::read_csv(ks_file, col_types = readr::cols(.default="d", Features="c"))
```

Comment:
`(readr::cols(.default="d", Features="c")))`

What do the letters mean in this case?

Reply:
d refers to `double` and c refers to `character`. It is similar to the `dtype` options that pandas has for columns. R does a better job at handling data types more explicitly than python!